plugin-maven/src/test/java/com/diffplug/spotless/maven/MavenRunner.java

```diff
@@ -135,28 +139,4 @@ public String toString() {
 		}
 	}
 
-	private static class Slurper extends Thread {
-		private final InputStream input;
-		private volatile String result;
-
-		Slurper(InputStream input) {
-			this.input = Objects.requireNonNull(input);
-			start();
-		}
-
-		@Override
-		public void run() {
-			try {
-				ByteArrayOutputStream output = new ByteArrayOutputStream();
-				ByteStreams.copy(input, output);
-				result = output.toString(Charset.defaultCharset().name());
-			} catch (Exception e) {
-				result = Throwables.getStackTraceAsString(e);
-			}
-		}
-
-		public String result() {
-			return result;
-		}
-	}
```

Comment:
I would revert this commit (c0879e11fb81861f9a702a35e1a9111eddb048d8).  If a process prints a lot to `stderr`, but nothing to `stdout` (as happens in some failure modes), it's possible for the error pipe to get full such that child process blocks when it tries to print to stderr, and then the child process will never exit.  That's why we slurp on separate threads.

Reply:
Yeah, I see. That was the first thing I did which helped with maven runner blocking. That might no longer be needed - I will check that on my end.

Although one could argue, that reading stderr/stdout from separate threads seems overly complicated, seeing that no-one ever uses the "error" part in the `MavenRunner.Result`. How about leaving it like it is, but merging stdout/stderr together using `		builder.redirectErrorStream(true);` and only reading the stdout afterwards?

Reply:
I remember a lot of pain "why isn't this working!!??" which created the multithreaded Slurper.  `redirectErrorStream` might work, but the old one definitely worked, so I'm inclined not to change it.  Also helps imo to know if something came from `stdout` or `stderr`.

Reply:
Ahhh, the memories of past-me's pain :-) I totally understand that part...

I see no harm in reverting it, if the other change does the trick for fixing the ci part 👍